src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderRepository.java

```diff
@@ -34,6 +34,6 @@ default Optional<Offender> findOffenderByNomsId(String nomsId){
     Page<Offender> getOffendersWithImagesCapturedAfter(@Param("start") LocalDateTime start, Pageable pageable);
 
     @Lock(LockModeType.PESSIMISTIC_WRITE)
-    @Query("SELECT o from Offender o left join fetch o.bookings where o.nomsId = :nomsId")
+    @Query("SELECT o from Offender o left join fetch o.bookings where o.nomsId = :nomsId and o.id = o.rootOffenderId")
```

Comment:
this seems the most sensible thing to do here - this will grab us the primary offender record